example-scenegraph-sdk/source/source/Main.brs

```diff
@@ -19,14 +19,17 @@ sub Main(args as dynamic)
     options.logLevel = mparticleConstants().LOG_LEVEL.DEBUG
     options.apiKey = "REPLACE WITH API KEY"
     options.apiSecret = "REPLACE WITH API SECRET"
+    'OPTIONAL: For use with our data master feature
+    options.dataPlanId = "REPLACE WITH DATA PLAN ID"
+    options.dataPlanVersion = 1 'REPLACE WITH DATA PLAN VERSION
     
     'If you know the users credentials, supply them here
     'otherwise the SDK will use the last known identities
     identityApiRequest = {userIdentities:{}}
     identityApiRequest.userIdentities[mparticleConstants().IDENTITY_TYPE.EMAIL] = "user@example.com"
     options.identifyRequest = identityApiRequest
     
-    options.environment = mparticleConstants().ENVIRONMENT.FORCE_PRODUCTION
+    options.environment = mparticleConstants().ENVIRONMENT.FORCE_DEVELOPMENT
```

Comment:
just curious. any reason for this change?

Reply:
Just so you need to make less changes to see your data in the livestream.

Reply:
ah good point.  thx